src/utils/useHeaderAuthLinks.tsx

```diff
@@ -3,46 +3,55 @@ import { Button } from 'design-system-react';
 import type { ReactElement } from 'react';
 import { NavLink, useLocation } from 'react-router-dom';
 
-const AUTH_LINKS_EXCLUDED = new Set(['/', '/profile/complete', '/summary']);
+const AUTH_LINKS_EXCLUDED = new Set(['/']);
```

Comment:
Two small additions here from the sidebar on the unauthenticated landing page. No one ever clicks on them, but they're hiding over here:
<img width="1013" alt="Screenshot 2024-12-12 at 4 38 26 PM" src="https://github.com/user-attachments/assets/e2640b44-e9ef-4cb3-bc44-ed1368dcc97b" />

`/privacy-notice` and `/paperwork-reduction-act-notice`